### ISSUE TITLE
apiKey not used with Node on Windows and Server on Synology server

### ISSUE THREAD

**Issue by shankiflang** [HaveAGitGat/Tdarr#1017]
**Describe the bug**
I started my server using auth: true
I created an api key on the webUi.

Twot things :
- If I put the apiKey in a dockercompose file, it works
- If I put the apiKey in the Tdarr_Node_Config.json on windows I get :
```bash
[2024-05-31T22:45:12.277] [ERROR] Tdarr_Node - Error
    at Socket.onpacket (C:\Tdarr_Updater\Tdarr_Node\node_modules\socket.io-client\build\cjs\socket.js:485:29)
    at Emitter.emit (C:\Tdarr_Updater\Tdarr_Node\node_modules\@socket.io\component-emitter\index.js:143:20)
    at C:\Tdarr_Updater\Tdarr_Node\node_modules\socket.io-client\build\cjs\manager.js:240:18
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21) {
  data: {
    message: 'No auth token or API key provided',
    serverVersion: '2.19.01',
    serverError: 'authFailed',
    autoUpdate: false
  }
}
[2024-05-31T22:45:12.277] [ERROR] Tdarr_Node - No auth token or API key provided
```

 - OS: Windws

**Comment by HaveAGitGat** [comment]
Are you able to post the Node log on startup? There's most likely a syntax error in your config.

**Comment by shankiflang** [comment]
> Are you able to post the Node log on startup? There's most likely a syntax error in your config.

Yeah of course, 
here is the last logs :
```bash
[2024-06-01T09:59:25.061] [INFO] Tdarr_Node - C:/Tdarr_Updater/configs/Tdarr_Node_Config.json
[2024-06-01T09:59:25.082] [INFO] Tdarr_Node - {
  nodeName: 'home',
  serverIP: 'XX.XXX.XXX.XXX',
  serverPort: '8266',
  handbrakePath: '',
  ffmpegPath: '',
  mkvpropeditPath: '',
  pathTranslators: [ { server: '/mnt/videos/', node: 'Z:/' } ],
  logLevel: 'INFO',
  priority: -1,
  platform_arch_isdocker: 'win32_x64_docker_false',
  processPid: 27200,
  cronPluginUpdate: '',
  apiKey: 'XXXXXXXXX',
  nodeID: 'E1C2xRli8'
}
[2024-06-01T09:59:25.085] [INFO] Tdarr_Node - Config validation passed
[2024-06-01T09:59:25.206] [INFO] Tdarr_Node - version: 2.17.01
[2024-06-01T09:59:25.207] [INFO] Tdarr_Node - platform_arch_isdocker: win32_x64_docker_false
[2024-06-01T09:59:25.207] [INFO] Tdarr_Node - Starting Tdarr_Node
[2024-06-01T09:59:25.208] [INFO] Tdarr_Node - Preparing environment
[2024-06-01T09:59:25.208] [INFO] Tdarr_Node - Path translator: Checking Node path: Z:/
[2024-06-01T09:59:25.221] [INFO] Tdarr_Node - Path translator: Success: Node path can be accessed: Z:/
[2024-06-01T09:59:25.303] [ERROR] Tdarr_Node - Node connection error to: XX.XXX.XXX.XXX:8266
[2024-06-01T09:59:25.303] [ERROR] Tdarr_Node - Error
    at Socket.onpacket (C:\Tdarr_Updater\Tdarr_Node\node_modules\socket.io-client\build\cjs\socket.js:485:29)
    at Emitter.emit (C:\Tdarr_Updater\Tdarr_Node\node_modules\@socket.io\component-emitter\index.js:143:20)
    at C:\Tdarr_Updater\Tdarr_Node\node_modules\socket.io-client\build\cjs\manager.js:240:18
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21) {
  data: {
    message: 'No auth token or API key provided',
    serverVersion: '2.19.01',
    serverError: 'authFailed',
    autoUpdate: false
  }
}
[2024-06-01T09:59:25.305] [ERROR] Tdarr_Node - No auth token or API key provided
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - ---------------Binary tests start----------------
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - handbrakePath:C:/Tdarr_Updater/Tdarr_Node/assets/app/HandBrakeCLI.exe
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - ffmpegPath:C:/Tdarr_Updater/Tdarr_Node/node_modules/ffmpeg-static/ffmpeg.exe
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - mkvpropedit:C:/Tdarr_Updater/Tdarr_Node/assets/app/mkvpropedit/windows/mkvpropedit.exe
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - Binary test 1: handbrakePath working
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - Binary test 2: ffmpegPath working
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - Binary test 3: mkvpropeditPath working
[2024-06-01T09:59:25.841] [INFO] Tdarr_Node - ---------------Binary tests end-------------------

```

here is the complete config from Tdarr_Node_Config.json:

```json
{
  "nodeName": "home",
  "serverIP": "XX.XXX.XXX.XXX",
  "serverPort": "8266",
  "handbrakePath": "",
  "ffmpegPath": "",
  "mkvpropeditPath": "",
  "pathTranslators": [
    {
      "server": "/mnt/videos/",
      "node": "Z:/"
    }
  ],
  "logLevel": "INFO",
  "priority": -1,
  "cronPluginUpdate": "",
  "apiKey": "XXXXXXXXX"
}
```

**Comment by HaveAGitGat** [comment]
`version: 2.17.01`

Thanks, update the node to the same version as the server which is `2.19.01` and should work. Think that issue was fixed in 2.18.02.
